### ISSUE TITLE
LC0016 - ignores ShowCaption property

### ISSUE THREAD

**Issue by sit-zm** [StefanMaron/BusinessCentral.LinterCop#579]
LC0016 warning is shown even if ShowCaption = false for the group property on page.

![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/75423560/6664944a-02ab-4a99-be60-10d56aff1132)

Best,
Zoran M.

**Comment by Arthurvdv** [comment]
There's already an check in place for the `ShowCaption` property, so the rule shouldn't be raised.

I'm having trouble to reproduce this, because with the examples below the LC0016 diagnostic isn't raised.

Can you provide a small code sample to reproduce this bug?

```AL
page 50100 MyPage
{
    Caption = 'MyPage';

    layout
    {
        area(Content)
        {
            group(MyGroup)
            {
                ShowCaption = false;
            }
        }
    }
}

pageextension 50100 "Customer Card" extends "Customer Card"
{
    layout
    {
        addafter(General)
        {
            group(MyGroup)
            {
                ShowCaption = false;
            }

        }
    }
}
```

**Comment by sit-zm** [comment]
Hi Arthur,

The sample is below. You can test it out.

```
pageextension 50000 DYCEMyExtension extends "Warehouse Shipment"
{
    layout
    {
        addlast(Shipping)
        {
            group("DYCEShippingTo")
            {
                Caption = 'Shipping To';

                field("DYCE Ship-To"; Rec."Assigned User ID")
                {
                    ApplicationArea = All;
                    Caption = 'Ship-to';
                    ToolTip = 'Specifies the address that the products on the sales document are shipped to. Default (Sell-to Address): The same as the customer''s sell-to address. Alternate Ship-to Address: One of the customer''s alternate ship-to addresses. Custom Address: Any ship-to address that you specify in the fields below.';
                }
                group(DYCEShipToFields)
                {
                    ShowCaption = false;
                }
            }
        }
    }
}
```
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/75423560/b97091b4-5866-4887-b233-c1b72739468e)

DYCE is mandatory prefix but that does not change the issue.

Best,
Zoran M.

**Comment by pri-kise** [comment]
I don't receive the warning LC0016 in my local deveopment environment (preview version), but I receive it in our pipelines
e.g. for CueGroup

**Comment by Arthurvdv** [comment]
I'm having trouble to reproduce this. A copy of your example doesn't thrown an warning for me 🤔

![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/44637996/8bb495a6-0a44-42a5-ad6b-429d3c2f0af0)

Can you very you have the latest LinterCop.dll (0.30.19.0) version?

%userprofile%\.vscode\extensions\ms-dynamics-smb.al-13.0.1007491\bin\Analyzers

![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/44637996/bab857f0-908a-49bd-858c-49ab2141b68c)

**Comment by sit-zm** [comment]
Now it works OK.

I have the latest version BusinessCentral.LinterCop.dll
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/75423560/25d865a5-56d2-47be-a51c-a25e40e0adf0)
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/75423560/f8445f58-88d0-45f1-bba3-c1037ee06e51)
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/75423560/3e880072-b7b2-49d4-b718-ca37b931cd08)
